docs/airframe-rpc.md

```diff
@@ -210,6 +210,7 @@ Supported client types are:
 - __sync__: Create a sync HTTP client (ServiceSyncClient) for Scala (JVM)
 - __async__: Create an async HTTP client (ServiceClient) for Scala (JVM) using Future abstraction (`F`). The `F` can be `scala.concurrent.Future` or twitter-util's Future. 
 - __scalajs__:  Create an RPC client (ServiceClientJS)
+- __grpc-sync__: Create a gRPC blocking client (ServiceGrpcSyncClient)
```

Comment:
This option is subject to change because generating sync, async, and Future-based clients inside the same class would be possible.